pallets/identity/src/lib.rs

```diff
@@ -76,7 +76,7 @@
 #![recursion_limit = "256"]
 
 pub mod types;
-pub use types::{DidRecords as RpcDidRecords, DidStatus};
+pub use types::{DidRecords as RpcDidRecords, DidStatus, PermissionedCallOriginData};
 
```

Comment:
when an identity uses the `forwarded` function to proxy a call, the call name, and module name in  `pallets/permissions` should be updated.